src/language/generic.js

```diff
@@ -1,4 +1,6 @@
-    /**
+const Rainbow = require('../../dist/rainbow.js');
```

Comment:
```suggestion
const Rainbow = require('./rainbow.js');
```

I worry about mixing `dist` and `src`. Since we're doing more to `dist` now it might be best to update `src` and rebuild. Instructions on how to build are in the `README`. See [here](https://github.com/Khan/rainbow#building).